mintapi/api.py

```diff
@@ -95,12 +96,20 @@ def get_web_driver(email, password, headless=False, mfa_method=None,
         chrome_options.add_argument('disable-dev-shm-usage')
         chrome_options.add_argument('disable-gpu')
         # chrome_options.add_argument("--window-size=1920x1080")
+    if session_path is not None:
```

Comment:
I think it would be really great if this could be default behavior, as I imagine most folks would want it and it might be hard to discover. I would suggest picking some sensible default (home / user dir perhaps) but allowing it to be disabled. Ie, "--session-path=None" will disable this feature, but you can specify a directory and if you don't you get a sensible default.